Makefile

```diff
@@ -77,10 +77,31 @@ BPF_MESSAGE := "without BPF support"
 
 # BPF cannot currently be compiled on ARMv7 due to this bug: https://github.com/iovisor/gobpf/issues/272
 # ARM64 builds are not affected.
+with_bpf := no
 ifneq ("$(ARCH)","arm")
-ifneq ("$(wildcard /usr/include/bcc/libbpf.h)","")
+ifneq ("$(wildcard /usr/include/bpf/libbpf.h)","")
+with_bpf := yes
 BPF_TAG := bpf
 BPF_MESSAGE := "with BPF support"
+CLANG ?= $(shell which clang || which clang-10)
```

Comment:
CO-RE requires Clang 10+ so that's what buildbox installs. Here we look for regular clang or clang-10. This is not ideal for folks building from source outside of buildbox. However it would require a more elaborate search to look for a working clang.
Same logic is also used below for `clang-format` and `llvm-strip`